src/registry/charts/echarts-area-chart.tsx

```
@@ -2270,7 +2284,7 @@ export function EChartsAreaChart<TData extends Record<string, unknown>>({
     };
     raf = requestAnimationFrame(tick);
     return () => cancelAnimationFrame(raf);
-  }, [live, isLoading, loadingPoints, loadingData]);
+  }, [renderer, live, isLoading, loadingPoints, loadingData]);
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/src/registry/charts/echarts-area-chart.tsx:2287:6
  2285 |     raf = requestAnimationFrame(tick);
  2286 |     return () => cancelAnimationFrame(raf);
> 2287 |   }, [renderer, live, isLoading, loadingPoints, loadingData]);
       |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Cannot access ref value during render
  2288 |
  2289 |   // ── Legend overlay position ──────────────────────────────────────────────────
  2290 |   // Insets match the Recharts legend's breathing room inside the plot frame.

(react-hooks/refs)